keyboards/hackpad/readme.md

```diff
@@ -0,0 +1,31 @@
+# hackpad
+
+![hackpad](https://git.imaginaerraum.de/Purox/Hackpad/src/branch/main/pic.jpg)
```

Comment:
Dead link.

Reply:
Fixed by the newest commit.
Thank you for noticing!

Reply:
Still doesn't work.
![image](https://github.com/qmk/qmk_firmware/assets/4781841/7a01f3fd-ebd1-4eb2-9978-e29376b95867)

Reply:
Sorry for the annoyance.. Now the link points to the raw image, and the image is previewed.